x/concentrated-liquidity/position_test.go

```diff
@@ -1248,3 +1249,83 @@ func (s *KeeperTestSuite) TestSetPosition() {
 		}
 	}
 }
+
+func (s *KeeperTestSuite) TestGetAndUpdateFullRangeLiquidity() {
```

Comment:
Why is this not table-driven?

Reply:
Changed to table driven https://github.com/osmosis-labs/osmosis/pull/4884/commits/529678ba556df038a39e9d46a762888666e31f69